lambda-code/api-end-to-end-test/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "api-end-to-end-test",
+  "version": "1.0.0",
+  "main": "main.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "rm -rf dist",
+    "build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json",
+    "build:dev": "tsc --project tsconfig.json --watch",
+    "postbuild": "cp package.json dist/package.json && cp yarn.lock dist/yarn.lock && cd ./dist && yarn install --production"
+  },
+  "dependencies": {
+    "@aws-sdk/client-dynamodb": "3.799.0",
+    "@aws-sdk/client-lambda": "3.799.0",
+    "@aws-sdk/lib-dynamodb": "3.799.0",
+    "axios": "^1.9.0",
+    "jose": "^6.0.10"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.149",
+    "@types/node": "^22.15.3",
+    "tsc-alias": "^1.8.15",
+    "typescript": "^5.8.3"
+  },
+  "packageManager": "yarn@4.9.1"
```

Comment:
Was there any particular reason to go with Yarn 4 instead of yarn classic?  Nothing aginst Yarn 4 but wondering if all the other lambdas should also use the same version?

Reply:
I think this is something we discussed recently. You did ask why I did not use the newer version of Yarn on a previous lambda work. After seeing that the web application had already made the switch to Yarn 4 I decided to start using it too.

Since I have time this afternoon I think I can just go ahead and migrate other lambdas so that it gets included in this PR.

Reply:
I actually hit a couple of issues while trying to switch to Yarn 4 for the rest of our lambdas. I will tackle it in a different PR. While doing this switch I will also make sure all other lambdas (where it is relevant) use NodeJS 22 and the new file architecture where we add the code under `/src` + other changes I did while working on this new API end to end test lambda.